.nycrc

```diff
@@ -1 +1 @@
-{"reporter": ["lcov","text"], "exclude": ["src/tests", "src/database"]}
\ No newline at end of file
+{"reporter": ["lcov","text"], "exclude": ["src/tests", "src/database", "src/helpers/mailer/sendMailer.js"]}
```

Comment:
I did not mean to exclude it, What you can do make sure when you hit the API of sending resent link email. you don't send an email to the user. So from the test use a valid email and check if you can get the email in your inbox. @James-nyagatare